docs/content/guide/component-router.ngdoc

```diff
@@ -6,18 +6,17 @@
 # Component Router
 
 <div class="alert alert-info">
-**Deprecation Notice:** In an effort to keep synchronized with router changes in Angular 2, this implementation of the Component Router (ngComponentRouter module) has been deprecated and will not receive further updates.
-We are investigating backporting the Angular 2 Router to Angular 1, but alternatively, use the {@link ngRoute} module or community developed projects (e.g. [ui-router](https://github.com/angular-ui/ui-router)).
+**Deprecation Notice:** In an effort to keep synchronized with router changes in Angular, this implementation of the Component Router (ngComponentRouter module) has been deprecated and will not receive further updates.
+We are investigating backporting the Angular Router to AngularJS, but alternatively, use the {@link ngRoute} module or community developed projects (e.g. [ui-router](https://github.com/angular-ui/ui-router)).
```

Comment:
I felt that having them so close would be enough (along with our other communications about this that will happen). But I can add the word "new" in front of Angular to make it a bit clearer.